packages/DropdownMenu/src/LinkItem/LinkItem.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PropTypes from "prop-types";
+import LinkItemStyles from "../Item.styles";
+
+const { bool, node, string } = PropTypes;
+
+const propTypes = {
+  /** HTML for each LinkItem */
+  children: node.isRequired,
+  /** The url for the href */
+  link: string.isRequired,
+  /** Should the link open content in a new tab */
+  isExternal: bool,
+};
+
+const defaultProps = {
+  isExternal: false,
+};
+
+const LinkItem = props => {
+  const { children, isExternal, link, ...remainingProps } = props;
+
+  const linkItemProps = {
+    role: "menuitem",
+    ...remainingProps,
+  };
+
+  if (isExternal) {
+    linkItemProps.target = "_blank";
```

Comment:
`rel="noopener noreferrer"` should be added to the element too, please.

Reference: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
iiiinteresting... 🥇